civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -3,57 +3,76 @@
  * @file
  * Template to display a view as a table.
  *
- * - $title : The title of this group of rows.  May be empty.
- * - $header: An array of header labels keyed by field id.
- * - $caption: The caption for this table. May be empty.
- * - $header_classes: An array of header classes keyed by field id.
- * - $fields: An array of CSS IDs to use for each field id.
- * - $classes: A class or classes to apply to the table, based on settings.
- * - $row_classes: An array of classes to apply to each row, indexed by row
- *   number. This matches the index in $rows.
- * - $rows: An array of row items. Each row is an array of content.
- *   $rows are keyed by row number, fields within rows are keyed by field ID.
- * - $field_classes: An array of classes to apply to each field, indexed by
- *   field id, then row number. This matches the index in $rows.
+ * @var string $title
+ *   The title of this group of rows.  May be empty.
+ * @var array $header
+ *   An array of header labels keyed by field id.
+ * @var string $caption
+ *   The caption for this table. May be empty.
+ * @var array $header_classes
+ *   An array of header classes keyed by field id.
+ * @var array $fields
+ *   An array of CSS IDs to use for each field id.
+ * @var string $classes
+ *   A class or classes to apply to the table, based on settings.
+ * @var array $row_classes
+ *   An array of classes to apply to each row, indexed by row number.
+ * @var array $rows
+ *   An array of row items. Each row is an array of content. $rows are keyed by
+ *   row number, fields within rows are keyed by field ID.
+ * @var array $field_classes
+ *   An array of classes to apply to each field, indexed by field id, then row.
+ * @var array $field_attributes
+ *   Array of arrays of attributes indexed by field id and then row.
+ * @var string $attributes
+ *   Empty
```

Comment:
It wasn't included in the original list and I have no information on what it represents except that it appears to always be empty in this template. If you have an idea what it is I can change it.